src/earthdataVarinfo/handler.py

```diff
@@ -0,0 +1,70 @@
+import os
+import traceback
```

Comment:
I think `traceback` and `json` are currently unused imports and so can be removed here. Also, if you update to use `tempfile.TemporaryDirectory` as a context manager, you'll be able to remove `shutil` here, too.

It's probably worth running [pylint](https://pylint.readthedocs.io/en/latest/) and/or [pycodestyle](https://pycodestyle.pycqa.org/en/latest/) over this file to just check for small things like unused imports or linting issues, etc.